src/rules/no-gratuitous-expressions.ts

```diff
@@ -119,6 +119,11 @@ const rule: TSESLint.RuleModule<string, string[]> = {
   },
 };
 
+function isInsideJSX(context: TSESLint.RuleContext<string, string[]>): boolean {
```

Comment:
I think it would be better to move this function to the scope of the caller, so we don't need to write that complex type for the argument, and use `context` implicitly